Makefile

```diff
@@ -39,6 +39,9 @@ libcxl.o libcxl_sysfs.o : CFLAGS += -fPIC
 libcxl.so: $(LIBNAME)
 	ln -s $(LIBNAME) libcxl.so
 
+$(LIBSONAME): $(LIBNAME)
```

Comment:
strange to add those lines, the link is supposed to be created when ldconfig is called on $(LIBNAME)